packages/flutter/lib/src/rendering/proxy_box.dart

```diff
@@ -3513,7 +3526,21 @@ class RenderSemanticsGestureHandler extends RenderProxyBox {
   @override
   void describeSemanticsConfiguration(SemanticsConfiguration config) {
     super.describeSemanticsConfiguration(config);
-
+    if (_semanticsActionOverrides != null && _semanticsActionOverrides.isNotEmpty) {
+      final Map<CustomSemanticsAction, VoidCallback> customAction = <CustomSemanticsAction, VoidCallback>{};
+      for (final CustomSemanticsAction override in _semanticsActionOverrides) {
+        switch (override.action) {
+          case SemanticsAction.scrollUp:
+          case SemanticsAction.scrollDown:
+          case SemanticsAction.scrollLeft:
+          case SemanticsAction.scrollRight:
+            customAction[override] = () { };
+        }
+      }
+      if (customAction.isNotEmpty) {
+        config.customSemanticsActions = customAction;
```

Comment:
Shouldn't the custom actions only be added if they're valid and a handler is available for them?

Reply:
For regular customAction yes, but the overrides go a different code path that their handlers will not be called.